### PR TITLE
Adds research on transaction list by Alby

### LINKED ISSUE #926: Add Alby research on transaction lists
The Alby team did a survey on the use of transaction lists. They shared their findings in a [blog post](https://blog.getalby.com/rethinking-transaction-lists/). Let's add this to the [research](https://bitcoin.design/guide/resources/design-research/) and [activity](https://bitcoin.design/guide/daily-spending-wallet/activity/) pages.

### PR DESCRIPTION
**Pull request by GBKS**
Added a brief summary to the activity page, and a link on the research page.

Previews:
- [Activity](https://deploy-preview-929--bitcoin-design-site.netlify.app/guide/daily-spending-wallet/activity/#design-considerations)
- [Design research](https://deploy-preview-929--bitcoin-design-site.netlify.app/guide/resources/design-research/#independent-research)

Closes #926

### CODE REVIEW

File: guide/daily-spending-wallet/activity.md [hunk 1]
```diff
@@ -106,6 +106,8 @@ On a technical level, a [transaction]({{ '/guide/how-it-works/transactions/' | r
 
 ##### Design considerations
 
+[Research](https://blog.getalby.com/rethinking-transaction-lists/) by the team at [Alby](https://getalby.com) has indicated that most users rarely visit transaction list. The most common reasons to do so are to confirm payment status, budgeting and to analyze payment behavior. The biggest challenges are navigating through endless lists to find a specific transaction, and unavailable metadata leaving users guessing as to who the payment was sent to and what for.
```
Comment (pavlenex) [inline guide/daily-spending-wallet/activity.md:109]:
```suggestion
[Research](https://blog.getalby.com/rethinking-transaction-lists/) by the team at [Alby](https://getalby.com) has indicated that most users rarely visit a transaction list. The most common reasons to do so are to confirm payment status, budgeting and to analyze payment behavior. The biggest challenges are navigating through endless lists to find a specific transaction and unavailable metadata leaving users guessing who the payment was sent to and what for.	
```

**Comment by MoritzKa** [comment]
Great that we can contribute. Looks good for me.

File: guide/daily-spending-wallet/activity.md [hunk 2]
```diff
@@ -106,6 +106,8 @@ On a technical level, a [transaction]({{ '/guide/how-it-works/transactions/' | r
 
 ##### Design considerations
 
+[Research](https://blog.getalby.com/rethinking-transaction-lists/) by the team at [Alby](https://getalby.com) has indicated that most users rarely visit a transaction list. The most common reasons to do so are to confirm payment status, budgeting and to analyze payment behavior. The biggest challenges are navigating through endless lists to find a specific transaction and unavailable metadata leaving users guessing who the payment was sent to and what for.
```
Comment (sbddesign) [inline guide/daily-spending-wallet/activity.md:109]:
```suggestion
[Research](https://blog.getalby.com/rethinking-transaction-lists/) by the team at [Alby](https://getalby.com) has indicated that most users rarely visit a transaction list. The most common reasons to do so are to confirm payment status, budgeting, and to analyze payment behavior. The biggest challenges are navigating through endless lists to find a specific transaction and unavailable metadata, leaving users guessing who the payment was sent to and what for.	
```

**Review by sbddesign — CHANGES_REQUESTED** [review]
Grammar nits, but looks like a great addition otherwise.

**Review by sbddesign — APPROVED** [review]
👻

**Pull request merged**